collector/textfile.go

```diff
@@ -116,48 +241,19 @@ func (c *textFileCollector) parseTextFiles() []*dto.MetricFamily {
 		}
```

Comment:
We could avoid a loop in `convertMetricFamilies` and a slice here by just calling it `convertMetricFamily` (singular) and calling it for every metric family here.